src/lib/common/JsonHelper.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+*
+* Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms
+* of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+#include "common/JsonHelper.h"
+
+std::string toJsonString(const std::string& input)
+{
+  std::ostringstream ss;
+  ss << '"';
+  for (std::string::const_iterator iter = input.begin(); iter != input.end(); ++iter) {
```

Comment:
Sorry ...
The '{' should go right under the start of the blocks keyword:

    switch (x)
    {
    case a:
      xxx();
      break;

    case ...
    }

'case' is a special case. It's not code, just a label, part of the 'switch infrastructure'.

for:

    for ()
    {
       ccc();
    }